bfck

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-java -cp bin/ brainfuck.Main $@
+java -cp target/classes/ fr.unice.polytech.si3.miaou.brainfuck.Main $@
```

Comment:
More specifically we want `-Dexec.args="$*"`.

However maven prints some shit on the standard output when running the exec plugin and it does interfere with our application (especially because we throw exceptions and we want to get an error code out of them), so the exec plugin cannot be used.